developer-docs-site/docs/issues-and-workarounds.md

```diff
@@ -5,14 +5,107 @@ slug: "issues-and-workarounds"
 
 # Issues and Workarounds
 
-This page documents issues that frequently come up and the suggested workarounds. 
+This page provides workarounds and answers for issues and questions that frequently come up. 
 
 :::tip Help keep this page up-to-date
 If you found an issue that is not on this page, submit a [GitHub Issue](https://github.com/aptos-labs/aptos-core/issues). Make sure to follow the issue format used in this document. 
 :::
 
 ## Nodes
 
+### Invalid EpochChangeProof: Waypoint value mismatch
+
+#### Description
+
+Run a validator node and receive this error:
+
```

Comment:
This sentence is not ringing right... they never actually get to run the validator node. In the process of...they get this error. I will leave it to you to adjust the wording.

Reply:
Done.